tools/ReleaseTool/ReleaseCommand.cs

```diff
@@ -103,22 +108,21 @@ public int Run()
                 {
                     if (masterHasBranchEnforcement)
                     {
-                        Console.WriteLine("Re-adding branch protection...");
+                        Logger.Info("Re-adding branch protection...");
                         gitHubClient.AddAdminBranchEnforcement(gitHubRepo, options.MasterBranch);
                     }
                 }
                 
                 // Create release
                 var release = CreateRelease(gitHubClient, gitHubRepo, gitClient);
 
-                Console.WriteLine(string.Empty);
-                Console.WriteLine($"Release Complete! Release hash: {gitClient.GetHeadCommit().Sha}");
-                Console.WriteLine(string.Empty);
-                Console.WriteLine($"Draft release: {release.HtmlUrl}");
+                Logger.Info("Release Successful!");
+                Logger.Info("Release hash: {0}", gitClient.GetHeadCommit().Sha);
+                Logger.Info("Draft release: {0}", release.HtmlUrl);
```

Comment:
Similarly - if we are not in unattended mode, we could open the URL in the browser

Reply:
Done! :)